File: Array/java/FindEvenNumberDigits.java
package Array.java;

public class FindEvenNumberDigits {

    public static boolean isEven(int n) {

        String s = Integer.toString(n);

        return s.length() % 2 == 0 ? true : false;
    }


    public static int findNumbers(int[] nums) {
        int count = 0;

        for(int n : nums) {
            if(isEven(n)) count++;
        }

        return count;
    }

    public static void main(String[] args) {

        System.out.println(findNumbers(new int[]{12,345,2,6,7896}));
        System.out.println(findNumbers(new int[]{555,901,482,1771}));
    }

}

File: Array/javascript/findEvenNumberDigits.ts
function findNumbers(nums:number[]):number {
    let countEvenNumber:number = 0;

    for(const i of nums) {
        let numDigits = i.toString().length;
        if(numDigits % 2 == 0) countEvenNumber++;
    }

    return countEvenNumber;
}

console.log(findNumbers([20,30,40,100]));